samples/bookinfo/networking/egress-rule-google-apis.yaml

```diff
@@ -4,18 +4,10 @@ metadata:
   name: googleapis
 spec:
   hosts:
-    - "*.googleapis.com"
+    - www.googleapis.com
   ports:
     - number: 443
       name: https
       protocol: http
----
-apiVersion: networking.istio.io/v1alpha3
-kind: DestinationRule
-metadata:
-  name: googleapis
-spec:
-  host: "*.googleapis.com"
-  trafficPolicy:
-    tls:
-      mode: SIMPLE # initiates HTTPS when talking to www.google.com
\ No newline at end of file
+  location: MESH_EXTERNAL
+  resolution: DNS
```

Comment:
nit: Don't need DNS here I think

We are planning on cleaning up unneeded usages of DNS resolution in the samples and docs

Reply:
OK, I will remove it.

Reply:
@howardjohn I just checked, for TLS origination DNS is required for some reason. Note that www.googleapis.com is used as a destination in the virtual service. (The virtual service is required to rewrite port 80 to 443).

Reply:
Great, good to know. I'll have to look more into that. Let me know when this isn't a WIP anymore

Reply:
@howardjohn The PR is ready, please review it.